src/main/java/com/thebois/controllers/game/ColonyController.java

```diff
@@ -0,0 +1,39 @@
+package com.thebois.controllers.game;
+
+import com.thebois.controllers.IController;
+import com.thebois.models.beings.AbstractBeingGroup;
+import com.thebois.views.game.ColonyView;
+import com.thebois.views.game.IView;
+
+/**
+ * Collects data form the world and sends it to its corresponding view.
```

Comment:
typo

form -> from

Reply:
style

Actually, this one is also super generic, try rewriting it to be more specific to colony.

Reply:
btw, doesn't IDEA tell you about the typos? Or have you learnt to ignore that like me because of all the `'being', did you mean begin?` 😛